code-studio/package.json

```diff
@@ -39,7 +39,10 @@
     "build-css:dist": "npm run lint-css && npm run clean-css && scripts/build-css.js --dist",
     "build-js": "scripts/build-js.js",
     "build-js:dist": "npm run lint-js && npm test && npm run clean-js && scripts/build-js.js --dist && scripts/uglify-js.js",
-    "start": "scripts/build-js.js --watch"
+    "start": "scripts/build-js.js --watch",
+    "convert-scss-vars": "scripts/convert-scss-variables.js",
+    "prebuild-js": "npm run convert-scss-vars",
+    "prebuild-js:dist": "npm run convert-scss-vars"
```

Comment:
They're automatically run before `build-js` thanks to [npm magic](https://docs.npmjs.com/misc/scripts):

> Additionally, arbitrary scripts can be executed by running npm run-script <pkg> <stage>. Pre and post commands with matching names will be run for those as well (e.g. premyscript, myscript, postmyscript).